flare/otf.py

```diff
@@ -61,6 +64,11 @@ class OTF:
             "srun".
         dft_kwargs ([type], optional): Additional DFT arguments. Defaults
             to None.
```

Comment:
Unrelated, but would it be possible to add some more information about dft_kwargs to the docstring?

Reply:
Done deal.